scripts/gen-i18n.js

```diff
@@ -177,7 +177,9 @@ const trObj = {};
 for (const tr of translatables) {
     trObj[tr] = tr;
```

Comment:
Perhaps explicitly check that `tr` is not undefined, to avoid future problems?

Reply:
Well, not having an entry in inputTranslations is a legitimate situation. If `translatables` contains `undefined`, something's gone very screwy and if we were worried about this, I'd rather check it before it comes out of the function that screwed it up.

Reply:
True, but then you have to check it in multiple places rather than just here. AFAICT `translatables` includes the strings from the .js files, so if you have undefined here you definitely have a problem, and can throw an error in a fool-proof way. At least you know you have a problem, even if you don't know what caused it.

Compare with checking all the various branches where you add strings, which makes it possible to figure out what went wrong, but it's also possible to miss/forget checking in some situation.